### ISSUE TITLE
[AttributeError]: module `signal` has no attribute `SIGALRM`

### ISSUE THREAD

**Issue by LSYS** [in-rolls/indicate#1]
Hi, I have an import error. When I run

```
from indicate import transliterate
```

I get the following `AttributeError` that the module `signal` has no attribute `SIGALRM`:
```
 AttributeError: module 'signal' has no attribute 'SIGALRM' 
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Input In [2], in <cell line: 1>()
----> 1 from indicate import transliterate

File D:\ProgramData\Anaconda3\lib\site-packages\indicate\__init__.py:1, in <module>
----> 1 from .transliterate import hindi2english
      3 __all__ = ["hindi2english"]

File D:\ProgramData\Anaconda3\lib\site-packages\indicate\transliterate.py:4, in <module>
      1 import sys
      2 import argparse
----> 4 from .hindi2english import HindiToEnglish
      6 hindi2english = HindiToEnglish.transliterate
      9 def main(argv=sys.argv[1:]):

File D:\ProgramData\Anaconda3\lib\site-packages\indicate\hindi2english.py:24, in <module>
     20     raise TimeoutException("Timeout!", "")
     23 # Change the behavior of SIGALRM
---> 24 signal.signal(signal.SIGALRM, timeout_handler)
     27 class HindiToEnglish(Base):
     28     MODELFN = "data/model/hindi_to_english/saved_weights/"

AttributeError: module 'signal' has no attribute 'SIGALRM'

```

I'm running on local win10 OS and Python 3.9

**Comment by soodoku** [comment]
@rajashekar: https://stackoverflow.com/questions/8420422/python-windows-equivalent-of-sigalrm

Windows doesn't have it --- https://stackoverflow.com/questions/52779920/why-is-signal-sigalrm-not-working-in-python-on-windows

**Comment by rajashekar** [comment]
@LSYS - Thank you for raising this issue. 
This issue is fixed in indicate 0.1.0 - please get the latest version.
